### PR TITLE
[Diffusers] Set import to slow as env variable

### PR DESCRIPTION
**Pull request by patrickvonplaten**
As far as I understand these environment variables will then apply to all tests, but they really only matter for the diffusers tests as the name is very diffusers specific

### CODE REVIEW

**Review by mishig25 — APPROVED** [review]
lgtm!

File: .github/workflows/build_main_documentation.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: Build PR Documentation
 
+env:
+  DIFFUSERS_SLOW_IMPORT: yes
```
Comment (mishig25) [inline .github/workflows/build_main_documentation.yml:4]: double checking that `DIFFUSERS_SLOW_IMPORT=yes` means there will be no lazy loading. Correct?

Comment (patrickvonplaten) [reply .github/workflows/build_main_documentation.yml:4]: Yeah exactly

Comment (patrickvonplaten) [reply .github/workflows/build_main_documentation.yml:4]: https://github.com/huggingface/diffusers/pull/5176

Comment (patrickvonplaten) [reply .github/workflows/build_main_documentation.yml:4]: I can test then easily on this PR

**Pull request merged**